azure/Guidelines.md

```diff
@@ -1098,8 +1180,65 @@ While it may be tempting to use a revision/version number for the resource as th
 
 <a href="#condreq-etag-depends-on-encoding" name="condreq-etag-depends-on-encoding">:white_check_mark:</a> **DO**, when supporting multiple representations (e.g. Content-Encodings) for the same resource, generate different ETag values for the different representations.
 
+<a href="#substrings" name="substrings"></a>
```

Comment:
FWIW, for headers this is pointless. It's not what commonmark will use during rendering, which is what GitHub et. al. use.